app/Views/programstudi/update.php

```diff
@@ -0,0 +1,66 @@
+<?= $this->extend('layouts/page_layout') ?>
+<?= $this->section('content') ?>
+        <div class="right_col" role="main" style="min-height: 3790px;">
+          <div class="">
+            <div class="page-title">
+              <div class="title_left">
+                <h3>Program Studi</h3>
+              </div>
+            </div>
+
+            <div class="clearfix"></div>
+
+            <div class="row">
+              <div class="col-md-12 col-sm-12 col-xs-12">
+                <div class="x_panel">
+                  <div class="x_content">
+                  <?= validation_list_errors() ?>
+
+                  <?= form_open("programstudi/{$user['id']}/update"); ?>
```

Comment:
<?= form_open("programstudi/{$programstudi['id']}/update"); ?>

Reply:
Resolved by the latest commit